Cargo.toml

```diff
@@ -18,16 +18,16 @@ version = "3.0.1"
 
 [dependencies]
 actix-web = { version = "3.0", default-features = false }
-actix-service = { version = "1.0", default-features = false }
+actix-service = { version = "1.0.6", default-features = false }
```

Comment:
Does it mean, this would not work without version "1.0.6"?

How it would look like for someone, who somehow depends exactly on "1.0.5"?

Reply:
Specifying the minimum version ensures the insecure versions of the library are not usable.

In practice this won't be a problem since actix-web uses this spec. Though it does show up on the dependency checker as insecure for this lib.

Reply:
Someone trying to depend on "=1.0.5" is going to have issues regardless.

Reply:
At the end they still can switch to previous version if they really need it. Ok, thank you!